liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1826,7 +1829,18 @@ protected void doMigration() throws Exception {
 
             try {
                 if (COMMANDS.UPDATE.equalsIgnoreCase(command)) {
-                    liquibase.update(new Contexts(contexts), new LabelExpression(getLabelFilter()));
+                    try {
+                        liquibase.update(new Contexts(contexts), new LabelExpression(getLabelFilter()));
+                    } catch (MigrationFailedException e) {
+                        final Map<String, Object> argsMap = new HashMap<>();
+                        argsMap.put("changeLogFile", changeLogFile);
+                        argsMap.put("database", database);
+                        argsMap.put("liquibase", liquibase);
+                        argsMap.put("migrationFailedException", e);
+                        argsMap.put("rollbackOnError", rollbackOnError);
+                        final CommandScope liquibaseCommand = createLiquibaseCommand(database, liquibase, "internalRollbackChangesetsOnError", argsMap);
+                        liquibaseCommand.execute();
+                    }
```

Comment:
We handle this within https://github.com/liquibase/liquibase-pro/pull/683/files#diff-73679d2cc5ccd2f9ed02917b6a2c4ef1328c2e93ac1a680dca4f35377d643355R45-R70

Reply:
I'm thinking it might be good to do it here too, in case the user doesn't have the pro jar loaded.

Reply:
Excuse me for the late answer, somehow the notification went past me.
It makes sense, but I think there is a specific case where the user doesn't have the pro license key, but they set the flag to false and the CLI won't notify them they have to be a pro user to have the rights to actually set the --rollback-on-error flag.

Reply:
I am not totally following your scenario @dyadyushko . Here's what I'm thinking:

- the user only has the core jar loaded. they deleted the commercial jar for some reason.
- they try to use this flag
- they will get a CommandNotFoundException (or something like that), because Liquibase core is trying to call a command that doesn't exist.
- if the license check occurred here, they'd get a nice pretty error message saying "you need a pro key to use this flag" or whatever

Reply:
Oh, yeah, now I got it clear, thanks @StevenMassaro